OSDOCS-19637:Update the z-stream RNs for 4.14.65 (#111467)
openshift/openshift-docs: modules/zstream-4-14-65.adoc

// Module included in the following assembly:
//
// * release_notes/ocp-4-14-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="zstream-4-14-65-bug-fixes_{context}"]
= RHSA-2026:15091 - {product-title} {product-version}.65 image release, fixed issues, and security update

Issued: 13 May 2026

[role="_abstract"]
{product-title} release {product-version}.65 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2026:15091[RHSA-2026:15091] advisory. There are no RPM packages for this release.

[NOTE]
====
{product-title} release {product-version}.64 was not issued.
====

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.65 --pullspecs
----

> = Fixed issues

Reviewer: ```suggestion
[id = needs ID]
== Fixed issues
```
surprised this didn't error on the two L1 headings

> * Before this update, the `osDiskImage.source.id` property in the user-provisioned infrastructure ARM template was incorrectly placed. As a consequence, installation failures occurred. With this release, the `storageProfile.osDiskImage.source.id` property location in the user-provisioned infrastructure ARM template is corrected. As a result, the user-provisioned infrastructure ARM template installation on {azure-first} does not encounter invalid parameter errors. (link:https://redhat.atlassian.net/browse/OCPBUGS-79654[OCPBUGS-79654])

Reviewer: ```suggestion
* Before this update, the `osDiskImage.source.id` property in the user-provisioned infrastructure ARM template was incorrectly placed. As a consequence, installation on {azure-first} failed. With this release, the `storageProfile.osDiskImage.source.id` property location in the user-provisioned infrastructure ARM template is corrected. As a result, the user-provisioned infrastructure ARM template on {azure-first} installation succeeds. (link:https://redhat.atlassian.net/browse/OCPBUGS-79654[OCPBUGS-79654])
```
so verbose! renoa dangit
